### PR TITLE
added model feature

### PR DESCRIPTION
**Pull request by Samarth-5109**
added model component
Screenshot (49)
e

### CODE REVIEW

**Comment by prathameshpcodes** [comment]
looks good enough to merge

File: components/model/modal.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                <li class="list-item"><a href="">Navigation</a></li>
+                <li class="list-item"><a href="">Slider</a></li>
+                <li class="list-item"><a href="">Toast</a></li>
+               
+            </ul>
+            </div>
+            
+        </aside>
+
+        <section class="main-container">
```
Comment (sumeetyeola) [inline components/model/modal.html:47]: Can use <main> tag here instead of section

File: components/model/modal.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;1,100&family=Shippori+Antique&display=swap");
+@import url("../../styles/doc.css");
+
+.model-container {
```
Comment (sumeetyeola) [inline components/model/modal.css:4]: Spelling correction here- 'modal' instead of 'model'

File: components/model/modal.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                <li class="list-item"><a href="images.html">Image</a></li>
+                <li class="list-item"><a href="">Inputs</a></li>
+                <li class="list-item"><a href="">Lists</a></li>
+                <li class="list-item"><a href="">Navigation</a></li>
+                <li class="list-item"><a href="">Slider</a></li>
+                <li class="list-item"><a href="">Toast</a></li>
+               
+            </ul>
+            </div>
+
```
Comment (sumeetyeola) [inline components/model/modal.html:44]: Avoid unnecesary spacing

File: components/model/modal.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+                    <div class="response-btn">
+                      <button class="btn close-btn">Close</button>
+                      <button class="btn save-btn">Save changes</button>
+                      
+                    </div>
+                    
+                  </div>
+                
+
+
```
Comment (sumeetyeola) [inline components/model/modal.html:77]: Avoid unnecesary spacing

File: components/model/modal.html (hunk 1 above)
Comment (sumeetyeola) [inline components/model/modal.html:47]: Just a suggestion you can use main tag here instead of section and cannot find closing section tag.

**Review by sumeetyeola — CHANGES_REQUESTED** [review]

**Pull request merged**